src/routes/animal_photos.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const AnimalPhotosService = require('../services/animal_photos');
+
+const router = express.Router();
+
+/* Get all the photos based on the category ids */
+router.get('/', async (req, res) => {
+  const categoryIds = req.query.categoryIds ? req.query.categoryIds.split(',') : [];
```

Comment:
What could you change to make this endpoint "more RESTful"?

Reply:
Maybe Post endpoint so that I can send the category ids in the payload to filter the photos.